examples/sveltekit/src/routes/index.svelte

```diff
@@ -3,9 +3,11 @@
     const query = `
       query PostCategory {
         allPostCategories (sortBy: "title", order: DESC) {
-          _collection
-          _filename
-          _slug
+          _flatbread {
+            filename
+            collection
+            slug
+          }
```

Comment:
might this be more user-friendly as...
```suggestion
          _metadata {
            filename
            collection
            slug
          }
```

Then you wouldn't feel like you're accessing Flatbread internals when querying